charts/fluentd-elasticsearch/templates/service-account.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: {{ include "fluentd-elasticsearch.fullname" . }}
+  name: {{ include "fluentd-elasticsearch.serviceAccountName" . }}
```

Comment:
This property will be empty by default, this case has to be handled.
I propose to use `{{ include "fluentd-elasticsearch.fullname" . }}-sa` if `serviceAccountName` is empty.

Reply:
the problem i had was about the service account, i specified a service account which did not match the default one so that the service account access rights were bad because the clusterrolebinding uses fluentd-elasticsearch.serviceAccountName.

Changing my service account for the value in fluentd-elasticsearch.fullname fix the issue. I don't know which of the template/helper is the good one but for sure there are inconsistency between the service account configuration and the clusterrolebinding one.